src/syslog.rs

```diff
@@ -86,6 +88,7 @@ impl fmt::Display for Priority {
 /// The facility of a syslog message.
 ///
 /// See syslog man pages for information on their semantics.
+#[allow(missing_docs)]
```

Comment:
I find facility to be a really terrible name. I guess we keep it because it's the same used by `syslog`, it sort of makes sense. But I would still be more descriptive in the documentation.